docs/Authentication.md

```diff
@@ -701,11 +701,15 @@ const MyPage = () => {
 export default MyPage;
 ```
 
-If you call `useAuthenticated()` with a parameter, this parameter is passed to the `authProvider` call as second parameter. that allows you to add authentication logic depending on the context of the call:
+`useAuthenticated` accepts an options object as its only argument, with the following properties:
+- `enabled`: whether it should check for an authenticated user
```

Comment:
- `enabled`: whether it should check for an authenticated user (`true` by default)

Reply:
Make it a suggestion next time ;)
```suggestion
- `enabled`: whether it should check for an authenticated user (`true` by default)
```